zjit/src/hir.rs

```diff
@@ -10694,7 +10694,7 @@ mod opt_tests {
           v43:ObjectExact = ObjectAllocClass Object:VALUE(0x1008)
           PatchPoint MethodRedefined(Object@0x1008, initialize@0x1040, cme:0x1048)
           PatchPoint NoSingletonClass(Object@0x1008)
-          v47:NilClass = CCall initialize@0x1070, v43
+          v47:NilClass = Const Value(nil)
```

Comment:
OT: It's weird that this isn't DCE'd, since the return value of initialize is not semantically used. I guess it got caught in a FrameState somehow.